__tests__/app.test.js

```diff
@@ -197,6 +197,39 @@ describe('Testing for News app', () => {
    
   });
 
+// Ticket 9 Starts
+   describe("9. GET /api/articles/:article_id/comments", () => {
+    test("check an article object has 5 property", () => {
+      return request(app)
+        .get(`/api/articles/1/comments`)
+        .expect(200)
+        .then(({body:{ article }}) => {
```

Comment:
`Article` here is supposed to be an array, so this test should have failed. If it does, maybe iterate through and do your assertions in each loop? :)

Reply:
Fixed, thanks!